examples/Vagrantfile

```diff
@@ -4,10 +4,23 @@
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "ubuntu/trusty64"
+  config.vm.define "trusty" do |trusty|
+  	trusty.vm.box = "ubuntu/trusty64"
+  	trusty.vm.hostname = "trusty64"
 
-  config.vm.provision "ansible" do |ansible|
-    ansible.playbook = "site.yml"
-    ansible.sudo = true
+  	trusty.vm.provision "ansible" do |ansible|
+    	ansible.playbook = "site.yml"
+    	ansible.sudo = true
```

Comment:
Indentation in this file seems weird too. Might need to make sure your editor recognizes this file as Ruby source code.

```diff
diff --git a/examples/Vagrantfile b/examples/Vagrantfile
index a017d24..429149c 100644
--- a/examples/Vagrantfile
+++ b/examples/Vagrantfile
@@ -5,22 +5,22 @@ VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.define "trusty" do |trusty|
-       trusty.vm.box = "ubuntu/trusty64"
-       trusty.vm.hostname = "trusty64"
+    trusty.vm.box = "ubuntu/trusty64"
+    trusty.vm.hostname = "trusty64"
 
-       trusty.vm.provision "ansible" do |ansible|
-       ansible.playbook = "site.yml"
-       ansible.sudo = true
-       end
+    trusty.vm.provision "ansible" do |ansible|
+      ansible.playbook = "site.yml"
+      ansible.sudo = true
+    end
   end
 
   config.vm.define "xenial" do |xenial|
-       xenial.vm.box = "ubuntu/xenial64"
-       xenial.vm.hostname = "xenial64"
+    xenial.vm.box = "ubuntu/xenial64"
+    xenial.vm.hostname = "xenial64"
 
-       xenial.vm.provision "ansible" do |ansible|
-       ansible.playbook = "site.yml"
-       ansible.sudo = true
-       end
+    xenial.vm.provision "ansible" do |ansible|
+      ansible.playbook = "site.yml"
+      ansible.sudo = true
+    end
   end
 end
```